common/xbps-src/shutils/update_check.sh

```diff
@@ -140,6 +141,10 @@ update_check() {
                 hgsrhtname="$(printf %s "$url" | cut -d/ -f4,5)"
                 url="https://hg.sr.ht/$hgsrhtname/tags"
                 rx='/archive/(v?|\Q'"$pkgname"'\E-)?\K[\d\.]+(?=\.tar\.gz")';;
+            *codeberg.org*)
+                codebergname="$(printf %s "$url" | cut -d/ -f4,5)"
```

Comment:
codebergname should be declared local.
Better yet, there should be one variable, say, `siteurlname`, instead of sfname, lpname, and so on. Commit separately if you feel like.

Reply:
Unifying these is definitely the right way to go, done.